packages/libs/components/src/plots/VolcanoPlot.tsx

```diff
@@ -203,14 +258,19 @@ function VolcanoPlot(props: VolcanoPlotProps, ref: Ref<HTMLDivElement>) {
         <XYChart
           xScale={{
             type: 'linear',
-            domain: [xAxisMin, xAxisMax],
-            clamp: true, // do not render points that fall outside of the scale domain (outside of the axis range)
+            domain: [
+              xAxisMin - +showXMinTruncationBar * xTruncationBarWidth,
+              xAxisMax + +showXMaxTruncationBar * xTruncationBarWidth,
```

Comment:
What is the purpose of the `+` in `+showXMinTruncationBar` and `+showXMaxTruncationBar`?

Reply:
I was intending to succinctly extend the axis range only if the truncation bars were there, so i wanted the boolean `showXMinTruncationBar` to become 0 or 1 to help with that calculation. 

However, turns out that I can make `showXMinTruncationBar` and others numbers up earlier in the code and avoid all this `+` nonsense!

Reply:
I see! This is a little clever. Could you add a comment explaining that the value will be `0` or `1`, which will then determine if truncation width is added? Once that's done, I think it's good to merge!